app/src/utils/calculations.js

```diff
@@ -20,4 +20,20 @@ export function summation(list, field) {
   const reducer = (acc, item) => acc.add(item[field])
   const sum = list.reduce(reducer, new BN('0'))
   return sum
-}
\ No newline at end of file
+}
+
+export function secondsToMonths(seconds) {
+  const minutes = seconds / 60
+  const hours = minutes / 60
+  const days = hours / 24
+  const months = days / 30
+  return Math.round(months)
+}
+
+export function monthsToSeconds(months) {
+  const days = months * 30
+  const hours = days * 24
+  const minutes = hours * 60
+  const seconds = minutes * 60
+  return Math.round(seconds)
+}
```

Comment:
What do you think of declaring the total seconds in a month above and do the multiplication and division directly?


Like 
```
export const SECONDS_IN_A_MONTH = 2628000

---

export function secondsToMonths(seconds) {
  return Math.round(seconds / SECONDS_IN_A_MONTH)
}

export function monthsToSeconds(months) {
  return Math.round(months * SECONDS_IN_A_MONTH)
}

```

> Note that the avarage days in a month is ~30.42